pmd-java/src/test/java/net/sourceforge/pmd/lang/java/rule/design/donotextendjavalangerror/Error.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.design.donotextendjavalangerror;
+
+
+
+/*
```

Comment:
This should be a javadoc comment (`/**`). Please use a formatter to add stars on the left and remove the excessive whitespace all over.